h-include.js

```diff
@@ -58,21 +58,15 @@ var hinclude;
           }
         }
 
-        if (fragment) {
-          var nodeList = container.querySelectorAll(fragment);
+        var nodeList = container.querySelectorAll(fragment);
 
-          if (nodeList === 0) {
-            console.warning("Did not find fragment in response");
-            return;
-          }
-
-          var wrap = document.createElement('div');
-          wrap.appendChild(nodeList[0].cloneNode(true));
-
-          element.innerHTML = wrap.innerHTML;
-        } else {
-          element.innerHTML = req.responseText;
+        if (nodeList === 0) {
```

Comment:
I think this is meant to be

``` javascript
if (nodeList.length === 0)
```
If we use `node = querySelector()` above then this line could be 
``` javascript
if (!node)
```